pycbc/psd/__init__.py

```diff
@@ -204,24 +204,27 @@ def insert_psd_option_group(parser, output=True, include_data_options=True):
                                   "to you, then it is probably safe to "
                                   "ignore this option.")
     # Options for PSD variation
+    psd_options.add_argument("--psdvar-segment", type=float,
+                             metavar="SECONDS", help="Length of segment "
+                             "when calculating the PSD variability.")
     psd_options.add_argument("--psdvar-short-segment", type=float,
                              metavar="SECONDS", help="Length of short segment "
-                             "when calculating the PSD variability.")
+                             "for outliers removal.")
```

Comment:
The use of the psd variation segment is not just for removing outliers as far as I understand. So, please keep the content of the earlier help, eg `Length of short segment for outlier removal in PSD variability calculation`

Reply:
psdvar-short-segment is used just for outliers removal, while psdvar-segment is used as a window when calculating the psd variability. 
Anyway your correction is consistent, so thank you for that!